packages/shared-models/src/ymodels.ts

```diff
@@ -199,8 +199,9 @@ export class YFile
 export class YNotebook
   extends YDocument<models.NotebookChange>
   implements models.ISharedNotebook {
-  constructor() {
+  constructor(enableDocumentWideUndoRedo: boolean) {
```

Comment:
This should be an `IOptions` object so it can evolve easier

Reply:
Done. Thx @blink1073